src/components/DebugModeUI/PostHashDialog.tsx

```diff
@@ -10,7 +10,7 @@ import DialogTitle from '@material-ui/core/DialogTitle'
 import { blue } from '@material-ui/core/colors'
 import { useFriendsList } from '../DataSource/useActivatedUI'
 import { Avatar } from '../../utils/components/Avatar'
-import { Profile } from '../../database'
+import type { Profile } from '../../database'
```

Comment:
Does `type` modifier auto adding by prettier?

Reply:
No, I manually add it.
But any new automatic import generated by TypeScript language service will add the type modifier when needed.